ice/paper.py

```diff
@@ -177,4 +177,4 @@ def __str__(self):
         return "\n\n".join(str(p) for p in self.paragraphs)
 
     def dict(self, *args, **kwargs):
-        return super().dict(*args, **kwargs, exclude={"paragraphs"})
+        return super().dict(*args, **{**kwargs, "exclude": {"paragraphs"}})
```

Comment:
optional, I think dict union is a bit clearer:

```suggestion
        return super().dict(*args, **(kwargs | {"exclude": {"paragraphs"}))
```